person.rb

```diff
@@ -1,14 +1,15 @@
 require './corrector'
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
Getters and Setters for `@rentals` should be present

Reply:
I don't think we may need the setter method for rentals, because rentals are an array of Rentals objects, and whenever we create a new Rental to a specific book we will push a new item to the array.
but I Implemented it as you requested.